dc_utils/templates/404.html

```diff
@@ -3,26 +3,28 @@
     <head>
         <meta charset="utf-8">
         <title>
-            Page Not found
+            Page not found
         </title>
+        <link rel="stylesheet" href="http://dc-shared-frontend-assets.s3-website.eu-west-2.amazonaws.com/styles.css">
```

Comment:
Is there any docs anywhere on how things get into the `dc-shared-frontend-assets` bucket?

Reply:
Nope, I can write them in the dev handbook? It's just manually copied into the bucket. I guess we could publish there from the design-system repo?

Reply:
https://github.com/DemocracyClub/design-system/pull/145